Added release notes for 1.8.1 version (#407)
okteto/docs: versioned_docs/version-1.8/self-hosted/install/releases.mdx

id: releases
---

## 1.8.1
23 May, 2023

### Bugfixes

> - Fixed an issue deleting dev and preview environments when using custom okteto manifests. The destroy operation was not taken into account the manifest used during deployment and it wasn't executing the destroy commands defined in the manifest.

Reviewer: ```suggestion
- Fixed an issue affecting deletion of dev and preview environments when using custom Okteto manifests. The destroy operation did not evaluate the custom manifest used during deployment, which meant the destroy commands were not executed.
```

non-blocking suggestion

- Upgraded the Okteto CLI version to [2.15.3](https://github.com/okteto/okteto/releases/tag/2.15.3).

>   - Build optimization now takes into account environment variables used in build args to rebuild an image or not.

Reviewer: ```suggestion
  - Build optimization now considers environment variables used in build args to determine when to rebuild an image.
```

non-blocking suggestion
